lua/astrocommunity/motion/mini-surround/init.lua

```diff
@@ -7,16 +7,29 @@ return {
   { "machakann/vim-sandwich", enabled = false },
   {
     "echasnovski/mini.surround",
-    event = "User AstroFile",
+    keys = function(plugin, keys)
+      -- Populate the keys based on the user's options
+      local mappings = {
+        { plugin.opts.mappings.add, desc = "Add surrounding", mode = { "n", "v" } },
+        { plugin.opts.mappings.delete, desc = "Delete surrounding" },
+        { plugin.opts.mappings.find, desc = "Find right surrounding" },
+        { plugin.opts.mappings.find_left, desc = "Find left surrounding" },
+        { plugin.opts.mappings.highlight, desc = "Highlight surrounding" },
+        { plugin.opts.mappings.replace, desc = "Replace surrounding" },
+        { plugin.opts.mappings.update_n_lines, desc = "Update `MiniSurround.config.n_lines`" },
+      }
+      mappings = vim.tbl_filter(function(m) return m[1] and #m[1] > 0 end, mappings)
+      return vim.list_extend(mappings, keys)
+    end,
     opts = {
       mappings = {
-        add = prefix .. "a", -- Add surrounding in Normal and Visual modes
-        delete = prefix .. "d", -- Delete surrounding
-        find = prefix .. "f", -- Find surrounding (to the right)
-        find_left = prefix .. "F", -- Find surrounding (to the left)
-        highlight = prefix .. "h", -- Highlight surrounding
-        replace = prefix .. "r", -- Replace surrounding
-        update_n_lines = prefix .. "n", -- Update `n_lines`
+        add = "gza", -- Add surrounding in Normal and Visual modes
+        delete = "gzd", -- Delete surrounding
+        find = "gzf", -- Find surrounding (to the right)
+        find_left = "gzF", -- Find surrounding (to the left)
+        highlight = "gzh", -- Highlight surrounding
+        replace = "gzr", -- Replace surrounding
+        update_n_lines = "gzn", -- Update `n_lines`
```

Comment:
This change should be reverted.

Reply:
I don't understand. You need those lines in order for mini-surround to map the keys to the actions. No?

Reply:
They were already set previously

Reply:
We were just using a common `prefix` variable rather than hard coding each one with the same prefix. So we should stay with that.

Reply:
Oh, yes, I totally missed that. Will fix.